hieradata/nodes/dev-anthrax.yaml

```diff
@@ -0,0 +1 @@
+ocf::packages::postfix::manage: false
```

Comment:
i think using tagged classes is an antipattern; they're not particularly reliable (depend on parse order, etc.)

if we used a role pattern, that would be an ideal place to put it